modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -1,62 +1,36 @@
 [[procedure-module-guidelines]]
 = Procedure Module Guidelines
 
-// [bhardest] - The original structure of the content here confused me since it's presented as a cross between a procedure template and a description of what a procedure is. I've taken a stab at reorganizing it in a way that hopefully is more clear and more consistent with template_concept.adoc.
+The required parts of a procedure module are a procedure and its introduction. Optionally, the module can also include prerequisites and additional resources.
 
-A procedure module is a procedure written with numbered steps -- what a customer needs to do to accomplish a goal successfully.
+.Schema of a procedure module
+image::procedure-diagram.png[]
 
-// [bhardest] - Here's a stab at a procedure diagram. Like with the concept, this can be made more professional later.
+[[writing-the-introduction]]
+.Writing the Introduction
+The introduction is a short description of the procedure. For example, it can be a lead-in sentence or an infinitive phrase (_To extract the certificate: <steps>_). See also _The IBM Style Guide_ footnoteref:[ibm-style-guide,DERESPINIS, Francis, Peter HAYWARD, Jana JENKINS, Amy LAIRD, Leslie McDONALD, Eric RADZINKSI. _The IBM style guide: conventions for writers and editors_. Upper Saddle River, NJ: IBM Press/Pearson, c2012. ISBN 0132101300.] for details on introducing procedures.
 
-image::procedure-diagram.png[]
+The introduction typically provides context for the procedure, such as:
 
-[[writing-the-short-description]]
-.Writing the Short Description
-This paragraph explains why the user performs the procedure, sets the context of the procedure, and may explain or list special considerations specific to this procedure. Keep the information brief and focused on what is needed for this specific procedure. Suggested length is 1-3 sentences, but it can be longer if needed.
+* Who is the user performing the procedure
+* Why and where the user performs the procedure
+* Special considerations specific to the procedure
 
-To write an effective short description, include information about when users should perform the procedure and why it is necessary. If applicable, you can also include information about who should perform it and where it should be performed.
+Keep the information brief and focused on what the user needs for this specific procedure. Suggested length is 1 - 3 sentences, but it can be longer.
```

Comment:
Ah, ok, you're not misinterpreting, but IBM is being stupid in this case. The accepted typographical convention is to use "en" dashes without spaces for ranges. See, e.g. http://practicaltypography.com/hyphens-and-dashes.html

Now, it's true that AsciiDoctor doesn't support an easy rendering of en dashes (instead, it replaces "--" with an "em" dash, but it's a known issue, and it will hopefully be fixed one day: https://github.com/asciidoctor/asciidoctor/issues/1174